app/src/main/kotlin/com/softteco/template/ui/feature/profile/ProfileViewModel.kt

```diff
@@ -93,11 +101,33 @@ class ProfileViewModel @Inject constructor(
     }
 
     private fun onProfileChanged(profile: Profile) {
-        viewModelScope.launch(appDispatchers.ui) { profileRepository.cacheProfile(profile) }
-        profileState.value.run {
-            if (this is GetProfileState.Success) {
-                profileState.value = GetProfileState.Success(profile)
+        viewModelScope.launch(appDispatchers.ui) {
+            profileRepository.cacheProfile(profile)
```

Comment:
`profileRepository.cacheProfile(profile)` already saves the profile, and it does it to the non-encrypted store. 
The idea of the ticket is to update the actual way of keeping profile data from non-encrypted to encrypted.
So, I would expect just updating the implementation of already existing methods (like profileRepository.cacheProfile). We don't need to keep profile twice (encrypted and non-encrypted).